opensrp-app/src/main/java/org/smartregister/repository/EventClientRepository.java

```diff
@@ -63,6 +63,8 @@
     protected Table clientTable;
     protected Table eventTable;
 
+    private Set<String> formSubmissionIds;
```

Comment:
This should be returned in the method and should not be an instance variable so that its reclaimed after batchInsert completes